### PR TITLE
fix: #1248 - To use values.yaml only 

### PR DESCRIPTION
**Pull request by jnikhila**
## Updates as part of PR
- Changes for adding `values.yaml` file for updating the env variables.
- Changes to the installation page for moving the cluster configuration back to the prerequisite section.

## Checklist
I have:
- [x] run the content through Grammarly
- [x] linked to sample apps when relevant
- [x] added the meta description for each page in the PR
- [x] minimized the callouts and added only when necessary
- [x] added the `queryString` parameter to the Tabs (if used)
- [x] masked PII in images. For example, login credentials, account details, and more
- [x] added images only when necessary
- [x] deleted the images that are no longer used for the updated pages in the PR
- [x] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [x] used the `<figure/>` tag instead of a markdown representation for images
- [x] added the `<figcaption/>` tag to add a caption to the image
- [x] added the `alt` attribute in the `` tag
- [x] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [x] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx [hunk 1]
```diff
@@ -54,25 +39,30 @@ Follow the steps below to install Appsmith on your Kubernetes cluster:
...
-  4. Load the Appsmith chart repository with:
+  2. Load the Appsmith chart repository with:
   ```bash
   helm repo update
   ```
 
-  5. Run the below command to deploy Appsmith:
+  3. Run the below command to deploy Appsmith:
   ```bash
   helm install appsmith appsmith/appsmith
```
Comment (pratapaprasanna) [inline website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx:62]:
```
helm install appsmith appsmith/appsmith -n appsmith --create-namespace
```

File: website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx [hunk 2]
```diff
@@ -84,60 +74,33 @@ Follow the steps below to install Appsmith on your Kubernetes cluster:
...
+4. Load the Appsmith chart repository with:
 
 ```bash
 helm repo update
 ```
-6. Run the below command to deploy Appsmith:
+5. Run the below command to deploy Appsmith:
 
 ```bash
 helm install appsmith appsmith-ee/appsmith
```
Comment (pratapaprasanna) [inline website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx:94]:
```
helm install appsmith appsmith-ee/appsmith -n appsmith --create-namespace
```

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 3]
```diff
@@ -87,62 +87,58 @@ After making changes, be sure to **restart the docker containers** for the chang
...
+    APPSMITH_MAIL_FROM:"test@test.com"
+  ```
 
-<Tabs groupId="appsmithEditions" queryString="current-edition">
-<TabItem label="Community Edition" value="communityEdition"> 
+4. Update the values with:
 
-1. Update the parameter values in the `values.yaml` file under the `applicationConfig` section:
+  ```bash
+  helm upgrade appsmith appsmith/appsmith -f values.yaml
```
Comment (pratapaprasanna) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:114]: helm upgrade appsmith appsmith/appsmith -f values.yaml -n appsmith

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 4]
```diff
@@ -87,62 +87,58 @@ After making changes, be sure to **restart the docker containers** for the chang
...
   applicationConfig:
     #highlight-next-line
     APPSMITH_MAIL_FROM:"test@test.com"
   ```
 
-  2. Update the values with:
+4. Update the values with:
 
   ```bash
   helm upgrade appsmith appsmith-ee/appsmith -f values.yaml
```
Comment (pratapaprasanna) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:139]: `helm upgrade appsmith appsmith/appsmith -f values.yaml -n appsmith`

**Review by pratapaprasanna — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/setup/instance-configuration/README.mdx (hunk 3 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/README.mdx:114]: These changes are done as part of #1295. These will get merged to main with that PR.

File: website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx (hunk 1 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx]: merged main.

File: website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx (hunk 2 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx]: merged main

File: website/docs/getting-started/setup/instance-configuration/README.mdx (hunk 4 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-configuration/README.mdx]: updated.

**Review by danciaclara — APPROVED** [review]

**Review by danciaclara — DISMISSED** [review]

**Review by pratapaprasanna — APPROVED** [review]

**Pull request merged**